schema_based_on_diagram.sql

```diff
@@ -0,0 +1,41 @@
+CREATE TABLE patients (
+    id INT GENERATED ALWAYS AS IDENTITY,
+    name varchar,
+    date_of_birth date,
+    PRIMARY KEY (id)
+);
+CREATE TABLE treatments (
+    id INT GENERATED ALWAYS AS IDENTITY,
+    type varchar,
+    name varchar,
+    PRIMARY KEY (id)
+);
+CREATE TABLE medical_histories (
+    id INT GENERATED ALWAYS AS IDENTITY,
+    admitted_at TIMESTAMP,
+    patient_id INT,
+    status varchar,
+    FOREIGN KEY (patient_id) REFERENCES  patients(id),
+    FOREIGN KEY (id) REFERENCES treatments(id),
+    PRIMARY KEY (id)
+);
```

Comment:
- [ ] Please create a join table for the many-to-many relationships between `medical_histories` and `treatments`.
![image](https://user-images.githubusercontent.com/83803414/197849583-cadfb8fa-2fa1-40de-972c-caaacdda0150.png)